lib/fastlane/actions/version_from_last_tag.rb

```diff
@@ -0,0 +1,134 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      VERSION_FROM_LAST_TAG = :VERSION_FROM_LAST_TAG
+    end
+
+    # To share this integration with the other fastlane users:
+    # - Fork https://github.com/KrauseFx/fastlane
+    # - Clone the forked repository
+    # - Move this integration into lib/fastlane/actions
+    # - Commit, push and submit the pull request
+
+    class VersionFromLastTagAction < Action
+      def self.run(params)
+        version_to_return = "#{params[:default_version]}"
+        regex = "#{params[:regex]}"
+
+        # If there are remotes fetch tags
+        remote_count = sh("git remote show | wc -l")
+        if remote_count > 0.to_s
+          sh("git fetch --tags")
```

Comment:
<img width="692" alt="screen shot 2016-01-11 at 9 58 14 pm" src="https://cloud.githubusercontent.com/assets/6491743/12253561/70d0be5c-b8ae-11e5-8797-e58a7760de88.png">

The answer is :boom: :boom: :boom: no